signer/apk2/apk2.go

```diff
@@ -176,6 +189,19 @@ func (s *APK2Signer) SignFile(file []byte, options interface{}) (signer.SignedFi
 	return signer.SignedFile(signedApk), nil
 }
 
+// go 1.16 doesn't support generics.
+// TODO: Replace string by T in function signature once we use go 1.18+
+func InsertIntoSliceAtIndex(destination []string, element string, index int) []string {
```

Comment:
Not a blocker but no need for this to be a package public function.

Reply:
TIL the case of the first letter tells whether a function is publicly exposed or not! (I'm a super-n00b at Go 😁). Thanks for pointing that out. Fixed!

Reply:
Really? yuck -- I thought we learned not to do that with Fortran! :(